requirements-opt.txt

```diff
@@ -0,0 +1 @@
+tqdm==4.65.0
```

Comment:
@rly should optional requirements also be listed in `setup.py` or only strict requirements?

Reply:
Optional requirements can be set up in `setup.py` with the key:
```
    extras_require={
        "tqdm": ["tqdm>=4.41.0"],
    },
```